cmd/cloudFoundryDeploy.go

```diff
@@ -438,153 +364,12 @@ func getAppName(config *cloudFoundryDeployOptions) (string, error) {
-		}
-
-		modified, err := _replaceVariables(manifestFile, manifestVariables, manifestVariablesFiles)
-		if err != nil {
-			return "", []string{}, []string{}, errors.Wrap(err, "Cannot prepare manifest file")
-		}
-
-		if modified {
-			log.Entry().Infof("Manifest file '%s' has been updated (variable substitution)", manifestFile)
-		} else {
-			log.Entry().Infof("Manifest file '%s' has not been updated (no variable substitution)", manifestFile)
-		}
-
-		err = handleLegacyCfManifest(manifestFile)
-		if err != nil {
-			return "", []string{}, []string{}, errors.Wrapf(err, "Cannot handle legacy manifest '%s'", manifestFile)
-		}
-	}
-
-	return "blue-green-deploy", deployOptions, smokeTest, nil
-}
-
-// validateManifestVariablesFiles: in case the only provided file is 'manifest-variables.yml' and this file does not
-// exist we ignore that file. For any other file there is no check if that file exists. In case several files are
-// provided we also do not check for the default file 'manifest-variables.yml'
-func validateManifestVariablesFiles(manifestVariablesFiles []string) ([]string, error) {
-
-	const defaultManifestVariableFileName = "manifest-variables.yml"
-	if len(manifestVariablesFiles) == 1 && manifestVariablesFiles[0] == defaultManifestVariableFileName {
-		// we have only the default file. Most likely this is not configured, but we simply have the default.
-		// In case this file does not exist we ignore that file.
-		exists, err := fileUtils.FileExists(defaultManifestVariableFileName)
-		if err != nil {
-			return []string{}, errors.Wrapf(err, "Cannot check if file '%s' exists", defaultManifestVariableFileName)
-		}
-		if !exists {
-			return []string{}, nil
-		}
-	}
-	return manifestVariablesFiles, nil
-}
-
-func toParameterMap(parameters []string) (*orderedmap.OrderedMap, error) {
-
-	parameterMap := orderedmap.NewOrderedMap()
-
-	for _, p := range parameters {
-		keyVal := strings.Split(p, "=")
-		if len(keyVal) != 2 {
-			return nil, fmt.Errorf("Invalid parameter provided (expected format <key>=<val>: '%s'", p)
-		}
-		parameterMap.Set(keyVal[0], keyVal[1])
-	}
-	return parameterMap, nil
-}
-
-func handleLegacyCfManifest(manifestFile string) error {
-	manifest, err := _getManifest(manifestFile)
-	if err != nil {
-		return err
-	}
-
-	err = manifest.Transform()
-	if err != nil {
-		return err
-	}
-	if manifest.IsModified() {
-
-		err = manifest.WriteManifest()
-
-		if err != nil {
-			return err
-		}
-		log.Entry().Infof("Manifest file '%s' was in legacy format has been transformed and updated.", manifestFile)
-	} else {
-		log.Entry().Debugf("Manifest file '%s' was not in legacy format. No transformation needed, no update performed.", manifestFile)
-	}
-	return nil
-}
-
-func prepareCfPushCfNativeDeploy(config *cloudFoundryDeployOptions) (string, []string, []string, error) {
+func prepareCfPushCfNativeDeploy(config *cloudFoundryDeployOptions) (string, []string, error) {
```

Comment:
~What is the argument that was removed? deployType?~
It was the post action, why is that not needed anymore?